.github/workflows/assign_reviewers_new_contributors.yml

```diff
@@ -0,0 +1,80 @@
+name: Assign Reviewers on PR Creation for New Contributors
+
+on:
+  pull_request:
+    types: [opened]  
+
+jobs:
+  assign-reviewers:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Assign Reviewers and Check for Merged PRs
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { pull_request, repository } = context.payload;
+            const owner = repository.owner.login;
+            const repo = repository.name;
+            const prNumber = pull_request.number;
+            const username = pull_request.user.login;
+
+            // Fetch all closed PRs of the user
+            const userPRs = await github.pulls.list({
+              owner,
+              repo,
+              state: "closed",
+              per_page: 100,  
```

Comment:
If only checking for "not zero", then just get 5?

Reply:
done